templates/focus-ios.xliff

```diff
@@ -426,6 +426,14 @@
         <source>Show Search Suggestions?</source>
         <note>Title for search suggestions prompt</note>
       </trans-unit>
+      <trans-unit id="Settings.DailyUsagePing.Message" xml:space="preserve">
+        <source>This helps Mozilla to estimate active users.</source>
```

Comment:
@razvanlitianu The comment calls out that "Mozilla" should be a placeholder

Reply:
This is one of the edge cases where having Mozilla hard-coded makes sense (telemetry is sent to Mozilla or completely disabled). That's the criteria we use on desktop (another area is certificates).

In any case, the comment should be consistent.

Reply:
I am going to approve this, the comment will be updated later.

Reply:
updated the comment here - https://github.com/mozilla-mobile/firefox-ios/pull/24254 cc @razvanlitianu @Delphine